Lib/fontbakery/specifications/loca.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from fontbakery.callable import check
+from fontbakery.checkrunner import PASS, FAIL
+# used to inform get_module_specification whether and how to create a specification
+from fontbakery.fonts_spec import spec_factory # NOQA pylint: disable=unused-import
+
+spec_imports = (
+    ('.', ('shared_conditions', )),
+)
+
+@check(id='com.google.fonts/check/180')
```

Comment:
I'll merge this now, but could you please add an explicit rationale metadata entry for this check on a future PRs? It would be good to always describe the rationale of each check, for instance discussing known cases where a problem may surface if the check is not passing.

Reply:
also, feel free to use a dalton.maag domain for the check id and we don't have to stick to numeric ids anymore. A keyword is the preferred form nowadays.